yt_dlp/extractor/nytimes.py

```diff
@@ -59,204 +51,546 @@ def get_file_size(file_size):
                 subtitles = self._merge_subtitles(subtitles, m3u8_subs)
             elif ext == 'mpd':
                 continue
-            #     formats.extend(self._extract_mpd_formats(
-            #         video_url, video_id, format_id or 'dash', fatal=False))
             else:
                 formats.append({
                     'url': video_url,
                     'format_id': format_id,
                     'vcodec': video.get('videoencoding') or video.get('video_codec'),
                     'width': int_or_none(video.get('width')),
                     'height': int_or_none(video.get('height')),
-                    'filesize': get_file_size(video.get('file_size') or video.get('fileSize')),
+                    'filesize': self._get_file_size(video.get('file_size') or video.get('fileSize')),
                     'tbr': int_or_none(video.get('bitrate'), 1000) or None,
                     'ext': ext,
                 })
 
-        thumbnails = []
-        for image in video_data.get('images', []):
-            image_url = image.get('url')
-            if not image_url:
-                continue
-            thumbnails.append({
-                'url': 'http://www.nytimes.com/' + image_url,
-                'width': int_or_none(image.get('width')),
-                'height': int_or_none(image.get('height')),
-            })
-
-        publication_date = video_data.get('publication_date')
-        timestamp = parse_iso8601(publication_date[:-8]) if publication_date else None
-
-        return {
-            'id': video_id,
-            'title': title,
-            'description': video_data.get('summary'),
-            'timestamp': timestamp,
-            'uploader': video_data.get('byline'),
-            'duration': float_or_none(video_data.get('duration'), 1000),
-            'formats': formats,
-            'subtitles': subtitles,
-            'thumbnails': thumbnails,
-        }
-
-
-class NYTimesIE(NYTimesBaseIE):
-    _VALID_URL = r'https?://(?:(?:www\.)?nytimes\.com/video/(?:[^/]+/)+?|graphics8\.nytimes\.com/bcvideo/\d+(?:\.\d+)?/iframe/embed\.html\?videoId=)(?P<id>\d+)'
-    _EMBED_REGEX = [r'<iframe[^>]+src=(["\'])(?P<url>(?:https?:)?//graphics8\.nytimes\.com/bcvideo/[^/]+/iframe/embed\.html.+?)\1>']
-
-    _TESTS = [{
-        'url': 'http://www.nytimes.com/video/opinion/100000002847155/verbatim-what-is-a-photocopier.html?playlistId=100000001150263',
-        'md5': 'd665342765db043f7e225cff19df0f2d',
-        'info_dict': {
-            'id': '100000002847155',
-            'ext': 'mov',
-            'title': 'Verbatim: What Is a Photocopier?',
-            'description': 'md5:93603dada88ddbda9395632fdc5da260',
-            'timestamp': 1398631707,
-            'upload_date': '20140427',
-            'uploader': 'Brett Weiner',
-            'duration': 419,
-        }
-    }, {
-        'url': 'http://www.nytimes.com/video/travel/100000003550828/36-hours-in-dubai.html',
-        'only_matching': True,
-    }]
```

Comment:
These URLs still work. I know the extractor was broken, and I see that you removed/renamed/rewrote the method it was using to perform most of the extraction, but we still shouldn't remove the extractor or tests. Mark it as `_WORKING = False` and have it just return an `id`-only info dict for now

Reply:
I added this back into the extractor.